pavelib/quality.py

```diff
@@ -272,7 +272,12 @@ def run_jshint(options):
     jshint_report = jshint_report_dir / "jshint.report"
     _prepare_report_dir(jshint_report_dir)
 
-    jshint_directories = ["common/static/js", "cms/static/js", "lms/static/js"]
+    jshint_directories = [
```

Comment:
Agreed. It would be ideal to just find all JavaScript files under edx-platform, and then have a blacklist of certain directories that we want to skip over (vendor libraries primarily, I would think).

http://jshint.com/docs/cli/#ignoring-files